<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calPrmDialog</class>
 <widget class="QDialog" name="calPrmDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="setButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>410</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   font-family:simhei;
   color: rgb(86,93,113);
   font:bold;
   font:30px;</string>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="readButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>410</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   font-family:simhei;
   color: rgb(86,93,113);
   font:bold;
   font:30px;</string>
   </property>
   <property name="text">
    <string>读取</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_pCalItemPat">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>204</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="m_pModIndex">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>204</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>150</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>校正值:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>150</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>参考值:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>630</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   font-family:simhei;
   color: rgb(86,93,113);
   font:bold;
   font:30px;</string>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>模块选项:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>150</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>校正选项:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="m_pCalIndexPat">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>204</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>20</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>参数校准</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
